client/i18n/locales/english/intro.json

```diff
@@ -2106,30 +2192,50 @@
         ]
       },
       "manv": { "title": "241", "intro": [] },
-      "bnvw": { "title": "242", "intro": [] },
+      "quiz-javascript-searching-and-sorting-algorithms": {
+        "title": "JavaScript Searching and Sorting Algorithms Quiz",
```

Comment:
same here. I don't know if we need js in the title because we are covering mainly cs searching and sorting algorithms that works in any langauge

Reply:
✅ Will be added with next commit.